src/components/UI/Stepper/index.tsx

```diff
@@ -0,0 +1,49 @@
+import {FunctionComponent, ReactElement} from "react"
+import {ReactComponent as StepDotNextIcon} from "../../../assets/icons/step-dot-next.svg"
+import {ReactComponent as StepDotCurrentIcon} from "../../../assets/icons/step-dot-current.svg"
+import {ReactComponent as StepDotDoneIcon} from "../../../assets/icons/step-dot-done.svg"
+import "./styles.scss"
+
+const Stepper: FunctionComponent<{
```

Comment:
While the current "Expand DAO" design dropped usage of this component - we're gonna use it in the nearest future for other components.